scripts/release-manager/commands/stable-prs.js

```diff
@@ -131,7 +131,10 @@ module.exports = function(vorpal, app) {
             .then((richPRs) => {
               return richPRs.filter((pr) => {
                 if (!pr.merged_at) {
-                  this.log(`${chalk.yellow.bold('WARNING')} ${pr.html_url} was not merged, should have the milestone unset.`);
+                  this.log(
+                    `${chalk.yellow.bold('WARNING')} ${pr.html_url} was not merged, 
+                     should have the milestone unset.`
```

Comment:
This changes the logged text (adding empty space and line breaks in between "merged," and "should"). This isn't desirable. Can you split this text like you did above (in `scripts/release-manager/cli.js`)?